docs/how-to-guides/environment-setup-and-maintenance/isis-data-area.md

```diff
@@ -0,0 +1,185 @@
+</br>
+
+#### Mission Specific Data Areas
+
+If you are only working with a few missions, save space by downloading just those data areas (find your download command on the following list).
+
+??? abstract "Mission Names & Download Commands"
+
+    For versions of ISIS prior to ISIS 4.1.0, please use the `--legacy` flag.
+
+    | Mission | Command |
+    | ------ | ------ |
+    | Apollo 15 | `downloadIsisData apollo15 $ISISDATA` |
+    | Apollo 16 | `downloadIsisData apollo16 $ISISDATA` |
+    | Apollo 17 | `downloadIsisData apollo17 $ISISDATA` |
+    | Cassini | `downloadIsisData cassini $ISISDATA` | 
+    | Chandrayaan 1 | `downloadIsisData chandrayaan1 $ISISDATA` |
+    | Clementine 1 | `downloadIsisData clementine1 $ISISDATA` |
+    | Dawn | `downloadIsisData dawn $ISISDATA` |
+    | ExoMars | `downloadIsisData tgo $ISISDATA` |
+    | Galileo | `downloadIsisData galileo $ISISDATA` | 
+    | Hayabusa 2 | `downloadIsisData hayabusa2 $ISISDATA` |
+    | Juno | `downloadIsisData juno $ISISDATA` |
+    | Kaguya | `downloadIsisData kaguya $ISISDATA` |
+    | Lunar Orbiter | `downloadIsisData lo $ISISDATA` |
+    | Lunar Reconnaissance Orbiter | `downloadIsisData lro $ISISDATA` |
+    | Mars Exploration Rover  | `downloadIsisData mer $ISISDATA` |
+    | Mariner10  | `downloadIsisData mariner10 $ISISDATA` |
+    | Messenger | `downloadIsisData messenger $ISISDATA` |
+    | Mars Express  | `downloadIsisData mex $ISISDATA` |
+    | Mars Global Surveyor  | `downloadIsisData mgs $ISISDATA` |
+    | Mars Reconnaissance Orbiter  | `downloadIsisData mro $ISISDATA` |
+    | Mars Science Laboratory  | `downloadIsisData msl $ISISDATA` |
+    | Mars Odyssey  | `downloadIsisData odyssey $ISISDATA` |
+    | Near  | `downloadIsisData near $ISISDATA` |
+    | New Horizons  | `downloadIsisData newhorizons $ISISDATA` |
+    | OSIRIS-REx  | `downloadIsisData osirisrex $ISISDATA` |
+    | Rolo  | `downloadIsisData rolo $ISISDATA` |
+    | Rosetta  | `downloadIsisData rosetta $ISISDATA` |
+    | Smart1  | `downloadIsisData smart1 $ISISDATA` |
+    | Viking 1 | `downloadIsisData viking1 $ISISDATA` |
+    | Viking 2 | `downloadIsisData viking2 $ISISDATA` |
+    | Voyager 1 | `downloadIsisData voyager1 $ISISDATA` |
+    | Voyager 2 | `downloadIsisData voyager2 $ISISDATA` |
+
+-----
+
+## ISIS SPICE Web Service
+
+ISIS can retrieve the SPICE data over the internet with a web service.
+Using the ISIS SPICE Web Service significantly reduces the size 
+of the required downloads from the data area.
+
+??? warning "Not all operations are supported by the SPICE Web Service"
+
+    Some instruments require mission data to be present for radiometric calibration, which is not supported by the SPICE Web Server. Some programs that are designed to run an image from ingestion through the mapping phase do not have an option to use the SPICE Web Service. For information specific to an instrument, see the [ISIS Application Docs](https://isis.astrogeology.usgs.gov/Application/index.html).
```

Comment:
is would name MEX HRSC specifically not having support as well since that is a common one.